### DISCUSSION TITLE
Inviting tophf for an AMA on the Subreddit r/chrome_extensions

### DISCUSSION THREAD

**Discussion by 3xp11i** [violentmonkey/violentmonkey#2194] [General]
Hello @tophf,

I’m Prakhar, a creative web developer and browser extensions enthusiast.

`I couldn’t find another way to reach out, so I hope you don’t mind me using GitHub Discussions for this. If this isn’t the right approach, feel free to delete this message.`

I’m a moderator of the [r/chrome_extensions](https://www.reddit.com/r/chrome_extensions/) subreddit, and we’re on the lookout for some amazing people in the browser extension space to participate in an AMA session. While searching, I came across your work and realized that I’ve used your projects frequently—they're fantastic!

I am sure you must be having something great to share with the internet about the world of browser extensions, and the life of an extension developer.

I’d love to extend a kind invitation for you to join us for an AMA with our community members. If you’re interested, please [DM me on Reddit](https://www.reddit.com/user/prakhartiwari0/) or share your Reddit username here so I can reach out to you directly. We can then discuss a time that works best for you.

Thank you for considering this! I’m looking forward to the possibility of chatting.

Regards,
Prakhar

**Comment by tophf** [comment]
Not interested so far, but I'm [tophf](https://www.reddit.com/user/tophf/) at reddit, so let's continue there.

**Reply by 3xp11i** [reply]
Thanks for replying, I have DMed you there!
